requirements.txt

```diff
@@ -2,4 +2,4 @@ paho-mqtt
 PyYAML<=5.2,>=4.2b1
 enum34
 cerberus
-
+gpiod
```

Comment:
This should only be in the module's `REQUIREMENTS` tuple so we don't have to install all requirements for all modules when installing the main program.

Reply:
I've moved this out of global requirements.